project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.1.10-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.1.11-SNAPSHOT"
```

Comment:
For Z releases it looks like you don't change this version at all.

The job will strip off the `-SNAPSHOT`, cut a release, then bump the Z and re-add `-SNAPSHOT`: `1.1.10-SNAPSHOT` -> `1.1.10` -> release 1.1.10 -> `1.1.11-SNAPSHOT`

Take a peek at the git commit history for the various Z releases for examples